info.xml

```diff
@@ -14,8 +14,8 @@
     <url desc="Support">https://github.com/agileware/au.com.agileware/civiteams/issues</url>
     <url desc="Licensing">http://www.gnu.org/licenses/agpl-3.0.html</url>
   </urls>
-  <releaseDate>2023-11-08</releaseDate>
-  <version>1.2</version>
+  <releaseDate>2024-06-19</releaseDate>
+  <version>1.2.1</version>
```

Comment:
Sorry, can we make this 1.2.2? This version is actually in production.

Reply:
Done